solidity/contracts/middleware/libs/InterchainAccountMessage.sol

```diff
@@ -12,12 +12,19 @@ import {TypeCasts} from "../../libs/TypeCasts.sol";
  */
 library InterchainAccountMessage {
     using TypeCasts for bytes32;
+    using TypeCasts for address;
+
+    enum MessageType {
+        CALLS,
+        COMMITMENT
+    }
```

Comment:
discussed offline
we will have 3 message types
calls, commit, and reveal

Reply:
Added